src/Modal/styles.less

```diff
@@ -4,12 +4,17 @@
 // * @import (inline) '../../react-gemini-scrollbar/node_modules/gemini-scrollbar/gemini-scrollbar.css';
 
 & when (@modal-enabled) {
+  .modal-wrapper {
```

Comment:
I'm surprised this wasn't caught by the linter, but could we put a newline above `.modal-wrapper`?